src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/domain/domain.kt

```diff
@@ -22,6 +22,8 @@ class Email(value: String) : Validated<String>(value, notBlank)
 class PhoneNumber(value: String) : Validated<String>(value, notBlank)
 class PrisonName(value: String) : Validated<String>(value, notBlank)
 class PrisonId(value: String) : Validated<String>(value, notBlank, alphanumeric, 2.minLength, 6.maxLength)
+class CourtId(value: String) : Validated<String>(value, notBlank)
```

Comment:
Option to add extra validation rules here, just need to check the implementation in the court-register repo.  Although the swagger docs and DB field size were not consistent when I checked the prison-register :-(